modules/passkey/contracts/interfaces/ISafeSignerFactory.sol

```diff
@@ -10,6 +10,15 @@ import {P256} from "../libraries/P256.sol";
  * @custom:security-contact bounty@safe.global
  */
 interface ISafeSignerFactory {
+    /**
+     * @notice Emitted when a new signer is created.
+     * @param signer The signer address.
+     * @param x The x-coordinate of the public key.
+     * @param y The y-coordinate of the public key.
+     * @param verifiers The P-256 verifiers to use.
+     */
+    event Created(address signer, uint256 x, uint256 y, P256.Verifiers verifiers);
```

Comment:
When we discussed in the sprint planning, what fields to index was still up for discussion. I would like to see some justification for the choice to not index the `signer` here.

Reply:
Here: https://github.com/safe-global/safe-modules/issues/397#issuecomment-2084695579